pkg/pquerier/frames.go

```diff
@@ -483,6 +474,10 @@ func (dc *dataColumn) initializeData(dataType DType) {
 		dc.data = make([]int64, dc.size)
 	case FloatType:
 		dc.data = make([]float64, dc.size)
+		floats := dc.data.([]float64)
```

Comment:
Use array copy for better performance if this is frequent operation
Look at pkg/aggregate/iterator.go:366 for example

Reply:
It happens once for every metric when query specifically for raw data.
if you select `max(cpu), avg(cpu), sqr(cpu), cpu, avg(diskio), diksio` all in one query it will happen twice